index.html

```diff
@@ -17,8 +17,43 @@
 </head>
 <body>
   <header>
+
   </header>
   <main>
+    <div class="container">
+      <!-- Dont know whther or not the new entry container will be need to be hidden at any point, setting a div and a class for it.-->
+      <div class="newEntryContainer">
+        <div class="row">
+          <div class="column-full form-head">
+            <h4>Code Journal</h4>
+          </div>
+          <div class="row">
+            <h3 class="column-full align-left">New Entry</h3>
+          </div>
+          <form class="inputForm" action="">
+            <div class="row">
+              <div class="column-half">
+                <img class="image-input" src="./images/placeholder-image-square.jpg" alt="New Entry Placeholder">
+              </div>
+              <div class="column-half">
+                <div class="column-full">
```

Comment:
I would remove this div, the column half works as the container for the labels and inputs since on the smaller screen, the css will have column-half set to 100%